### PR TITLE
:bug: NetBox: Fix auth failure for PostgreSQL

### PR DESCRIPTION
**Pull request by nomaster**
When initializing NetBox, an authentication failure for the PostgreSQL
database connection occurs. Make sure to use the right username and
password.

### CODE REVIEW

File: charts/netbox/README.md [hunk 1]
```diff
@@ -204,7 +204,7 @@ The following table lists the configurable parameters for this chart and their d
 | `externalDatabase.username`                     | Username for external PostgreSQL                                    | `netbox`                                     |
 | `externalDatabase.password`                     | Password for external PostgreSQL (see also `existingSecret`)        | `""`                                         |
 | `externalDatabase.existingSecretName`           | Fetch password for external PostgreSQL from a different `Secret`    | `""`                                         |
-| `externalDatabase.existingSecretKey`            | Key to fetch the password in the above `Secret`                     | `postgresql-password`                        |
+| `externalDatabase.existingSecretKey`            | Key to fetch the password in the above `Secret`                     | `postgres-password`                        |
```
Comment (ghost) [inline charts/netbox/README.md]: Indentation error at the end of the table

**Review by ghost — APPROVED** [review]
It would add an new indentation error to the table but I approve it since the table is already messed up

File: charts/netbox/values.yaml [hunk 2]
```diff
@@ -313,7 +317,7 @@ externalDatabase:
   username: netbox
   password: ""
   existingSecretName: ""
-  existingSecretKey: postgresql-password
+  existingSecretKey: postgres-password
```
Comment (mraerino) [inline charts/netbox/values.yaml]: should this maybe default to `password`?

Comment (nomaster) [reply charts/netbox/values.yaml]: Would look cleaner, but let's not change the API here

**Review by mraerino — APPROVED** [review]

**Pull request merged**